src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -285,76 +331,117 @@ BinaryCompactObject::BinaryCompactObject(
 }
 
 Domain<3> BinaryCompactObject::create_domain() const {
-  const double inner_sphericity_A = object_A_.is_excised() ? 1.0 : 0.0;
-  const double inner_sphericity_B = object_B_.is_excised() ? 1.0 : 0.0;
+  // Get useful information about the type of grid used around each compact
+  // object
```

Comment:
Probably easier to store these in private member variables than to  recompute them